src/core/ext/transport/chttp2/server/chttp2_server.cc

```diff
@@ -1144,3 +1187,46 @@ void grpc_server_add_channel_from_fd(grpc_server* /* server */, int /* fd */,
 }
 
 #endif  // GPR_SUPPORT_CHANNELS_FROM_FD
+
+void grpc_server_accept_connected_endpoint(
+    grpc_core::ListenerInterface* core_listener,
+    std::unique_ptr<grpc_event_engine::experimental::EventEngine::Endpoint>
+        endpoint) {
+  core_listener->AcceptConnectedEndpoint(std::move(endpoint));
```

Comment:
I think we need to declare an `ExecCtx` here.

Reply:
Good catch.